### ISSUE TITLE
"Get orbital elements" dialog fails to show

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#2084]
I built the master (8bca4a5), and I can't get the "Get orbital elements" dialog in the SSE to show up. When pressing the button, the progress bar to the lower right shows, but no dialog (where you select objects to add) shows up.

![goe](https://user-images.githubusercontent.com/37028368/144701005-88cf78b9-ff39-4878-a7fd-22d4f87b2df1.PNG)

**Comment by gzotti** [comment]
I only touched the comets part of the SSE recently. Is there something in the logfile?

**Comment by Atque** [comment]
I found these two lines in the log:

> qt.network.ssl: QSslSocket::connectToHostEncrypted: TLS initialization failed
Download error: While downloading "https://www.minorplanetcenter.net/iau/Ephemerides/Comets/Soft00Cmt.txt" the following error occured: "TLS initialization failed"

I have discovered that, for some reason, I can't access all internet based services when I build myself. The public beta works. I guess I do something wrong when building from source.
